java/food-packaging/src/main/java/org/acme/foodpackaging/domain/LineOperatorUpdatingVariableListener.java

```diff
@@ -0,0 +1,45 @@
+package org.acme.foodpackaging.domain;
+
+import ai.timefold.solver.core.api.domain.variable.VariableListener;
+import ai.timefold.solver.core.api.score.director.ScoreDirector;
+import org.jspecify.annotations.NonNull;
+
+import java.util.Objects;
+
+public class LineOperatorUpdatingVariableListener implements VariableListener<PackagingSchedule, Line> {
+    @Override
+    public void beforeVariableChanged(@NonNull ScoreDirector<PackagingSchedule> scoreDirector, @NonNull Line line) {
+        // Empty method
```

Comment:
We can see that. :-) Don't say _what_, say _why_ - such as "No need to do anything."